gorush/notification_apns.go

```diff
@@ -22,10 +22,13 @@ func InitAPNSClient(key string) (*apns.Client, error) {
 			LogError.Errorf("Key %s key_password not exist", key)
 			return nil, errors.New("APNS key_password not exists")
 		}
+		path = PushConf.Ios.KeyMap[key]
+		password = PushConf.Ios.KeyMap[key]
```

Comment:
`password = PushConf.Ios.KeyMap[key]` needs to be changed to `password = PushConf.Ios.KeyPass[key]`